packages/Logger/src/index.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * MIT License
+
+ * Copyright (c) 2018 Priit Kallas
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+**/
+
+/**
+ * This interface heavily borrows from the `ts-log` package. However, the `trace`
+ * function is not on all of the common loggers that we claim to support so
+ * it's been removed in this copy of this interface.
+ */
+
+/**
+ * Represents a generic logger that could be a simple console, bunyan etc.
+ */
+export interface Logger {
```

Comment:
It's a type in AS fwiw.  Does that matter?

Reply:
Not really (but we just so happened to talk about this today)